src/qt/pivx/forms/addresseswidget.ui

```diff
@@ -386,12 +390,16 @@
              <item>
               <widget class="QLabel" name="labelAddress">
                <property name="text">
-                <string>Enter address</string>
+                <string>Enter PIVX address</string>
```

Comment:
kinda redundant to be saying "Enter PIVX address" from the PIVX wallet. We can leave it as "Enter address".